specification/communication/data-plane/CallAutomation/preview/2023-10-03-preview/communicationservicescallautomation.json

```diff
@@ -1745,6 +1745,12 @@
             "schema": {
               "$ref": "#/definitions/RecordingStateResponse"
             }
+          },
+          "202": {
+            "description": "The service has accepted the StartCallRecording request and will begin processing it. You will receive either a StartRecordingFailed event or RecordingStateChanged to active at your specified callback URI to update you on the status of the request.",
```

Comment:
May not be active if started in Paused mode, need to change the verbose

Reply:
We need definition for 'StartRecordingFailed' as well here.

Reply:
Updated description, and added definition